docs/source/changes.rst

```diff
@@ -4,9 +4,15 @@ Changelog
 v2.10 (in development)
 ----------------------
 
+Improvements and Changes:
+
+- Examples README
```

Comment:
Let's just have one bullet point in the "Improvements and Changes" section that talks about this PR and everything it does. Make sure to end it with "(gh-820)" like the examples below (the lifted_gate one is an exception and needs to be fixed.